react-nginx/README.md

```diff
@@ -0,0 +1,122 @@
+## Details
+
+An project deploy a React Application with Nginx
+
+### Command
+
+Build image and run container
+
+```
+$ docker-compose up --build -d  --rm 
+```
+
+Stop and remove the container
+
+```
+$ docker-compose down
+```
+
+### Structure
+
+This base project created with [Create React App](https://github.com/facebook/create-react-app).
+
+```
+├── docker-compose.yml
+├── Dockerfile
+├── .dockerignore
+├── .gitignore
+├── .nginx
+│   └── nginx.conf
+├── package.json
+├── public
+│   ├── favicon.ico
+│   ├── index.html
+│   ├── logo192.png
+│   ├── logo512.png
+│   ├── manifest.json
+│   └── robots.txt
+├── README.md
+├── node_modules
+├── src
+│   ├── App.css
+│   ├── App.js
+│   ├── App.test.js
+│   ├── index.css
+│   ├── index.js
+│   ├── logo.svg
+│   ├── reportWebVitals.js
+│   └── setupTests.js
+└── yarn.lock
+
+```
```

Comment:
Sorry to insist, but we take care that all the READMEs of the samples keep the same presentation pattern.
Can you check existing ones and follow the order to describe your sample please?
ie: 
https://github.com/docker/awesome-compose/tree/master/react-java-mysql
https://github.com/docker/awesome-compose/tree/master/nextcloud-postgres